modules/monitoring-specifying-how-a-service-is-monitored-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="specifying-how-a-service-is-monitored-for-cluster-observability-operator_{context}"]
+= Specifying how a service is monitored for Cluster Observability Operator
+
+To use the metrics exposed by your sample service, you must configure monitoring components to scrape metrics from the `/metrics` endpoint. 
+
+You can create this configuration by using a `ServiceMonitor` custom resource definition (CRD) that specifies how the service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. 
+The former requires a `Service` object, while the latter does not, which enables Prometheus to scrape metrics directly from the metrics endpoint exposed by a pod.
```

Comment:
```suggestion
The `ServiceMonitor` object requires a `Service` object, while the `PodMonitor` object does not. This enables Prometheus to scrape metrics directly from the metrics endpoint exposed by a pod.
```

I'd spell this out rather than using latter, just because we don't use words like "respectively" and I feel like this is in the same vein. Also, I'd make this sentence shorter if possible by splitting it.

Reply:
Good suggestion. Made the sentences shorter and clarified the meaning, I hope,